packaging/mlops-agent.spec

```diff
@@ -234,39 +234,39 @@ install -m 0755 packaging/run-unittest.sh %{buildroot}%{_bindir}/tizen-unittests
 %postun -p /sbin/ldconfig
 
 %files
-%manifest machine-learning-agent.manifest
+%manifest mlops-agent.manifest
 %license LICENSE
-%attr(0755,root,root) %{_bindir}/machine-learning-agent
-%attr(0644,root,root) %{_unitdir}/machine-learning-agent.service
-%attr(0644,root,root) %config %{_sysconfdir}/dbus-1/system.d/machine-learning-agent.conf
+%attr(0755,root,root) %{_bindir}/mlops-agent
+%attr(0644,root,root) %{_unitdir}/mlops-agent.service
+%attr(0644,root,root) %config %{_sysconfdir}/dbus-1/system.d/mlops-agent.conf
 %attr(0644,root,root) %{_datadir}/dbus-1/system-services/org.tizen.machinelearning.service.service
 
-%files -n libmachine-learning-agent
-%manifest machine-learning-agent.manifest
+%files -n libmlops-agent
+%manifest mlops-agent.manifest
 %license LICENSE
-%{_libdir}/libml-agent.so.*
+%{_libdir}/libmlops-agent.so.*
 
-%files -n libmachine-learning-agent-devel
-%manifest machine-learning-agent.manifest
-%{_libdir}/libml-agent.so
-%{_libdir}/libml-agent.a
-%{_includedir}/ml-agent/ml-agent-interface.h
-%{_libdir}/pkgconfig/ml-agent.pc
+%files -n libmlops-agent-devel
+%manifest mlops-agent.manifest
+%{_libdir}/libmlops-agent.so
+%{_libdir}/libmlops-agent.a
+%{_includedir}/mlops-agent/ml-agent-interface.h
+%{_libdir}/pkgconfig/mlops-agent.pc
 
 %files test
-%manifest machine-learning-agent.manifest
+%manifest mlops-agent.manifest
 %license LICENSE
-%attr(0755,root,root) %{test_base_dir}/machine-learning-agent-test
+%attr(0755,root,root) %{test_base_dir}/mlops-agent-test
```

Comment:
need fix 'test_base_dir'

%define test_base_dir %{_bindir}/mlops-agent-test
or
%define test_base_dir %{_bindir}/ml

Reply:
I am going to use `ml`. Thanks

Reply:
https://github.com/nnstreamer/deviceMLOps.MLAgent/blob/ae046961f187d8adf50d7daa9d562c8c858930be/packaging/mlops-agent.spec#L16